packages/cli/src/lib/commands/init/processes/generate-lh-budgets.unit.test.ts

```diff
@@ -0,0 +1,54 @@
+import { handleBudgetsGeneration } from './generate-lh-budgets';
+import { RcJson } from '@push-based/user-flow';
+import deriveHelpers = require('../derive-budgets-from-lhr');
+import fileHelpers = require('../../../core/file');
+
+jest.mock('../derive-budgets-from-lhr');
+jest.mock('../../../core/file');
+
+describe('generate LH budgets', () => {
+
+  beforeEach(() => {
+    jest.resetAllMocks();
```

Comment:
In general, I recommend using `clear` because `restore` and `reset` do some additional stuff to mocked function implementations/returned values which might disrupt the tests.

```suggestion
    jest.clearAllMocks();
```